core/templates/dev/head/player/PlayerServices.js

```diff
@@ -302,6 +307,43 @@ oppia.factory('oppiaPlayerService', [
     }
   };
 
+  var _loadInitialInformationCardData = function(successCallback) {
+
+      var infoCardDataUrl = '/createhandler/data/' +  _explorationId;
```

Comment:
Nit: extra space before _explorationId.

Also, add:

    // TODO(sll): move these three handlers into a separate exp_metadata.py controller.

(since now they're being used by both the player and the editor).